### DISCUSSION TITLE
Navigating NAS folders with no cover art

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1578] [Fixed Issues]
Only recently started happening. Not sure which update was the first to show it, but all recent updates (Dec23/Jan24) seem to have the same behaviour.

Browse into a NAS folder containing only music files (no image files present). If the music files do not contain embedded cover art and the matching cover art cannot be found on the interwebs (maybe the artist is too obcure - I listen to a lot of '70s '80s folk artists) then rAudio seems to get stuck in a loop searching for cover art. The rA+ logo at top left keeps 'flickering' and it is almost impossible to navigate back out of the folder.

I can fix the issue by dropping a folder.jpg (usually stolen from Discogs) into the folder. rAudio stops looping as soon as it picks up the jpg and I can then navigate calmly into other folders. Small problem, but annoying...

**Comment by somax80** [comment]
Same problem for me also on navigating  usb drive.

It seems to infinite loop on:
POST to  mpdlibrary.php
GET assets/img/coverart.svg?v=1708190553
GET assets/img/coverart.svg?v=1708190553

the php script execute continously bash script:  /srv/http/bash/status-coverart.sh
and top command highlight process kid3-cli been running continously.

copying a folder.jpg file in impacted folder, stops the loop.

rAudio 20240212
6.1.64-v8+ aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 908 MiB
4 x Cortex-A53 @ 1.20 GHz
